tdb/vidrl_upload.py

```diff
@@ -28,116 +29,120 @@ def read_vidrl(path, fstem, assay_type):
     '''
     Read all csv tables in path, create data frame with reference viruses as columns
     '''
-    fname = path + fstem + ".csv"
     exten = [ os.path.isfile(path + fstem + ext) for ext in ['.xls', '.xlsm', '.xlsx'] ]
 
     if True in exten:
         ind = exten.index(True)
-        convert_xls_to_csv(path, fstem, ind)
-        fname = "data/tmp/%s.csv"%(fstem)
-        parse_vidrl_matrix_to_tsv(fname, path, assay_type)
+        convert_vidrl_xls_to_tsv(path, fstem, ind, assay_type)
     else:
         print("Unable to recognize file {}/{}".format(path,fstem))
         sys.exit()
 
-def convert_xls_to_csv(path, fstem, ind):
-    ref_sera_row=10    # Should be <= serum_strain_row_index
-    titer_col_start=4 # Should be <= start_col
+def convert_vidrl_xls_to_tsv(path, fstem, ind, assay_type):
     import xlrd
```

Comment:
consider relocating this to the standard `import` block at the top of the file, [per PEP8 guidance](https://peps.python.org/pep-0008/#imports)

Reply:
Fixed in https://github.com/nextstrain/fauna/pull/157/commits/3efd0fe38035f884ad03e3e42bddd430a0172fbe